### PR TITLE
Update service.md

### PR DESCRIPTION
**Pull request by ponypaver**
adjust grammar

### CODE REVIEW

**Comment by ponypaver** [comment]
I signed it

File: content/en/docs/concepts/services-networking/service.md [hunk 1]
```diff
@@ -356,7 +356,7 @@ that are configured for a specific IP address and difficult to re-configure.
 The IP address that you choose must be a valid IPv4 or IPv6 address from within the
 `service-cluster-ip-range` CIDR range that is configured for the API server.
 If you try to create a Service with an invalid clusterIP address value, the API
-server will returns a 422 HTTP status code to indicate that there's a problem.
+server will return a 422 HTTP status code to indicate that there's a problem.
```
Comment (sftim) [inline content/en/docs/concepts/services-networking/service.md]: Good spot!

With the [style guide](https://kubernetes.io/docs/contribute/style/style-guide/) in mind, can I suggest:
```suggestion
server returns a 422 HTTP status code to indicate that there's a problem.
```

Comment (sftim) [reply content/en/docs/concepts/services-networking/service.md]: That's essentially a nit, and this is also OK to merge as is, so:
/lgtm

**Comment by poothia** [comment]
lgtm

**Pull request merged**